plugin-flex-ts-template-v2/src/feature-library/ring-notification/helpers/audioPlayerManagerHelper.ts

```diff
@@ -0,0 +1,25 @@
+import * as Flex from '@twilio/flex-ui';
+
+import { getFeatureFlags } from '../../../utils/configuration';
+
+class AudioPlayerManagerHelper {
+  _mediaId: string | undefined;
+
+  play = (): void => {
+    const custom_data = getFeatureFlags() || {};
+    this._mediaId = Flex.AudioPlayerManager.play({
+      url: `https://${custom_data.serverless_functions_domain}/features/ring-notification/phone_ringing.mp3`,
```

Comment:
The URL doesn't work when running `localhost`. If you look at the `chat-to-video` feature's `SwitchToVideo` component you can see an example that works in all scenarios. Could you please adjust the logic here to be the same?